divi/src/txdb.h

```diff
@@ -8,7 +8,9 @@
 #define BITCOIN_TXDB_H
 
 #include "leveldbwrapper.h"
-#include "main.h"
+#ifndef BITCOIN_COINS_H
+#include <coins.h>
+#endif
```

Comment:
double header guard

Reply:
Also, do you need coins.h for a concrete type? If not better to just forward declare it and move the inclusion into the source

Reply:
fixed it